bazel_tools/haskell-haskell-lsp-binary-fp.patch

```diff
@@ -0,0 +1,47 @@
+diff --git a/haskell-lsp-types/src/Language/Haskell/LSP/Types/Uri.hs b/haskell-lsp-types/src/Language/Haskell/LSP/Types/Uri.hs
```

Comment:
In case you didn’t look at the linked PR. This fixes the Binary instance to roundtrip empty file paths. This matters for incremental builds where we serialize shake keys.